binding/python/examples/theano/cnn.py

```diff
@@ -40,7 +40,7 @@
 from theano.tensor.nnet.conv import conv2d
 from theano.tensor.signal.downsample import max_pool_2d
 import multiverso as mv
-from multiverso.theano_ext.sharedvar import mv_shared
+from multiverso.theano_ext.sharedvar import mv_shared, sync_all_mv_shared_vars
```

Comment:
According to Google Python style guide, we should
- only import packages or modules rather than specific class. [1]
- import should be in separate lines so that to avoid modify same line multiple times in git.

[1] https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports
[2] https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting